docs/developer/architecture/centralized-cli.rst

```diff
@@ -0,0 +1,172 @@
+Centralized CLI
+===============
+
+Context
+-------
+
+MetalK8s comes with a set of services to operate and monitor the K8s cluster.
+All operations that need to be performed by the Platform Administrator could be
+categorized as follow:
+
+- Cluster Resources Administration (Nodes, Volumes, Deployments, ...)
```

Comment:
What are *Deployments* in this context?

Reply:
Deployments means Expansions ?

Reply:
I was referring to Deployment Object on K8s API

Reply:
That's kinda confusing because MetalK8s doesn't really 'care' that much about (K8s) *Deployments*. I'd suggest another term ('cluster services'?)

Reply:
Maybe 'addons'?

Reply:
Actually, I am talking about any K8s resource we can create, edit, delete through k8s apiserver. I suggest k8s resource

Reply:
But this should be left to `kubectl` as we said... Do you still want to mention this in the requirements?